pepys_import/resolvers/command_line_resolver.py

```diff
@@ -97,26 +97,50 @@ def resolve_datafile(self, data_store, datafile_name, datafile_type, privacy, ch
     def resolve_platform(
         self, data_store, platform_name, platform_type, nationality, privacy, change_id
     ):
-        options = [f"Search for existing platform", f"Add a new platform"]
+        platform_details = []
+        final_options = [f"Search for existing platform", f"Add a new platform"]
         if platform_name:
-            options[1] += f", default name '{platform_name}'"
+            # If we've got a platform_name, then we can search for all platforms
+            # with this name, and present a list to the user to choose from,
+            # alongside the options to search for an existing platform and add
+            # a new platform
+
+            # The order-by clause is important, to get the same ordering of
+            # options on different platforms/db backends, so that our tests work
+            platforms = (
+                data_store.session.query(data_store.db_classes.Platform)
+                .join(data_store.db_classes.Nationality)
+                .filter(data_store.db_classes.Platform.name == platform_name)
+                .order_by(
+                    data_store.db_classes.Platform.identifier.asc(),
+                    data_store.db_classes.Nationality.name.asc(),
```

Comment:
We are ordering Nationalities by their priorities when we resolve Nationality. Do you think that it could be better for users if we use the same order here?

Reply:
Good idea - I'll do that now!

Reply:
Fixed in 86b05a4